cpp/src/neighbors/ivf_pq/ivf_pq_search.cuh

```diff
@@ -137,6 +149,18 @@ void select_clusters(raft::resources const& handle,
       alpha = -1.0;
       beta  = 0.0;
     } break;
+    case cuvs::distance::DistanceType::CosineExpanded: {
+      alpha = -1.0;
+      beta  = 0.0;
+
+      auto float_queries_matrix_view =
+        raft::make_device_matrix_view<float, uint32_t>(float_queries, n_queries, dim_ext);
+
+      raft::linalg::row_normalize(handle,
```

Comment:
I feel like normalizing the queries twice shouldn't be necessary. As long as we use the `qc_distances` only for selecting the clusters for one query, its norm shouldn't affect search results at all. You normalize the rotated queries further below, that should be enough.

Reply:
Done. We do not normalize when computing qc_distances.